libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -390,7 +390,7 @@ bytes BinaryTransform::operator()(BuiltinCall const& _call)
 		return toBytes(Opcode::Unreachable);
 	else if (_call.functionName == "nop")
```

Comment:
Shouldn't all these files use `_yulstring` ?

Reply:
`YulString` was designed for the optimizer where you don't really care about the contents of the string but mostly only check for equality and want a quick way to sort that is also efficient if you have many long strings that share the same long prefix.

The binary transform usually only looks at each string once and wants to know its actual value.